src/Components/Nav/NavSort.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import DotCMSApi from '../../libs/dotcms.api';
+import PageContext from '../../PageContext';
+
+class NavSort extends Component {
+    static contextType = PageContext;
+
+    emitEvent(siteId, pageUrl) {
+        const url = [
+            `/c/portal/layout?p_l_id=b7ab5d3c-5ee0-4195-a17e-8f5579d718dd&p_p_id=site-browser`,
```

Comment:
Make this `b7ab5d3c-5ee0-4195-a17e-8f5579d718dd` a const with like: `PORTLET_ID`